digits/model/images/classification/job.py

```diff
@@ -57,3 +57,6 @@ def download_files(self, epoch=-1):
 
         return download_files
 
+    @override
+    def parent_job(self):
+        return [self.train_task().dataset]
```

Comment:
why don't you move this up one level in the class hierarchy i.e. in `digits/model/images/job.py`? That would save the hassle of duplicating the implementation in `digits/model/generic/job.py`